### DISCUSSION TITLE
New install of birdnet pi incorrectly identifying birds

### DISCUSSION THREAD

**Discussion by ghost** [mcguirepr89/BirdNET-Pi#1113] [General]
hello, just installed birdnet and it's identifying birds from Africa.  I configured my coordinate and set my timezone.  Please help.

**Comment by Svardsten53** [comment]
Be sure you are using "BirdNET_GLOBAL_6K_V2.4_Model_FP16 (2023)" for Model. This will use an eBird filter to remove unlikely species based on location and time of the year.

Don't set Minimum Confidence value too low, start with 0,75 and see what happens.

Believe it or not, but a similar thing happened to another person in my country. He got a number of exotic species. After changing the Privacy Threshold to 1% the error disappeared! I have no idea why.
